lib/class-wp-theme-json-gutenberg.php

```diff
@@ -206,7 +206,8 @@ class WP_Theme_JSON_Gutenberg {
 	 *              `--wp--style--root--padding-*`, and `box-shadow` properties,
 	 *              removed the `--wp--style--block-gap` property.
 	 * @since 6.2.0 Added `outline-*`, and `min-height` properties.
-	 * @since 6.3.0 Added `writing-mode` property.
+	 * @since 6.4.0 Added `writing-mode` property.
```

Comment:
Should this be 6.3 still?

Reply:
It is somewhat unrelated to this PR, but 6.3 is incorrect; it was never included in 6.3.